guides/assets/javascripts/guides.js

```diff
@@ -1,53 +1,53 @@
-$.fn.selectGuide = function(guide) {
-  $("select", this).val(guide);
-};
-
-var guidesIndex = {
-  bind: function() {
-    var currentGuidePath = window.location.pathname;
-    var currentGuide = currentGuidePath.substring(currentGuidePath.lastIndexOf("/")+1);
-    $(".guides-index-small").
-      on("change", "select", guidesIndex.navigate).
-      selectGuide(currentGuide);
-    $(document).on("click", ".more-info-button", function(e){
-      e.stopPropagation();
-      if ($(".more-info-links").is(":visible")) {
-        $(".more-info-links").addClass("s-hidden").unwrap();
-      } else {
-        $(".more-info-links").wrap("<div class='more-info-container'></div>").removeClass("s-hidden");
-      }
-    });
-    $("#guidesMenu").on("click", function(e) {
-      $("#guides").toggle();
-      return false;
+(function() {
+  "use strict";
+  window.syntaxhighlighterConfig = { autoLinks: false };
+
+  this.wrap = function(elem, wrapper) {
+    elem.parentNode.insertBefore(wrapper, elem);
+    wrapper.appendChild(elem);
+  }
+
+  this.unwrap = function(elem) {
+    var wrapper = elem.parentNode;
+    wrapper.parentNode.replaceChild(elem, wrapper);
+  }
+
+  this.createElement = function(tagName, className) {
+    var elem = document.createElement(tagName);
+    elem.classList.add(className);
+    return elem;
+  }
+
+  document.addEventListener("DOMContentLoaded", function() {
+    var $guidesMenu = document.getElementById("guidesMenu");
```

Comment:
`$` in variable names is generally used to indicate that they're jQuery collections / objects.